src/nodedef.cpp

```diff
@@ -752,15 +750,15 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 	}
 
 	if (is_liquid) {
+		// Vertex alpha is no longer supported, correct if necessary.
+		correctAlpha(tdef, 6);
+		correctAlpha(tdef_spec, CF_SPECIAL_COUNT);
```

Comment:
Why doing that for liquids only? Are they the only nodes which support the `alpha` parameter?